src/AdminSettings.vue

```diff
@@ -30,9 +30,9 @@
 					class="checkbox"
 					:disabled="loading"
 					@change="saveData">
-				<label for="admin_approval">{{ t('registration', 'Require admin approval') }}</label>
+				<label for="admin_approval">{{ t('registration', 'Require administrator approval') }}</label>
```

Comment:
Should we use "administration" like in the readme and info.xml?

Reply:
Yes you are right 😉.